datasets/meetbouten/meetbouten.json

```diff
@@ -193,6 +193,8 @@
         "$schema": "http://json-schema.org/draft-07/schema#",
         "type": "object",
         "additionalProperties": false,
+        "mainGeometry": "geometrie",
+        "identifier": "identificatie",
```

Comment:
Dit veranderd API, toch? Is dataportaal klaar voor dit?

Reply:
Ha Nick, goed punt. Ik heb het nu zo gehouden dat als dit veld niet aanwezig is in het schema als default gewoon "geometry" en "id" worden gebruikt. Dus alleen voor nieuwe datasets zou dit kunnen gaan afwijken, maar dan moeten we toch in overleg met Dataportaal. Als het dataportaal idd gaat lukken om obv. het schema stub-code te genereren voor de frontend (zoals we pas met Jon Koops bespraken), zou dat helemaal mooi zijn, want dan kunnen ze gelijk rekening gaan houden met de "mainGeometry" en "identifier".